docs/CHANGELOG.asciidoc

```diff
@@ -157,24 +159,30 @@ When you upgrade to 8.0.0, use the following API changes:
 
 * Use `service_tokens` instead of `service-tokens`.
 
-* `check-permissions` is no longer supported.
```

Comment:
I removed this because it's not a deprecation, and I did not see it documented anywhere in older API docs.

Reply:
right, we haven't changed this now, and it is planned to be removed in a future release